wandb/integration/keras/keras.py

```diff
@@ -563,7 +563,7 @@ def set_params(self, params):
         self.params = params
 
     def set_model(self, model):
-        self.model = model
+        super().set_model(model)
```

Comment:
yea, i found this one too, but there was another issue somewhere, something about "_inner_layers" or something that wasn't as straightforward to fix.

Reply:
yep, that's why i added this: https://github.com/wandb/wandb/pull/8241/files#diff-e5f76827d21d0d5ba6baf3ff760189d9f3e76d3cc1d141e3d3a3d5223e6cb237R25-R26